lib/Net/DAVTalk.pm

```diff
@@ -112,6 +114,7 @@ sub new {
   }
 
   my $Self = bless \%Params, ref($Class) || $Class;
+  $Self->{SSL_options} = delete $Params{SSL_options};
```

Comment:
oh actually i think this line is unnecessary, since $Self is a blessed \%Params anyway.  so this might just be the one line fix below